AlphaWallet/Accounts/ViewModels/AccountsViewController.swift

```diff
@@ -21,19 +21,16 @@ class AccountsViewController: UIViewController {
     private let config: Config
     private let keystore: Keystore
     private let balanceCoordinator: GetNativeCryptoCurrencyBalanceCoordinator
-    private let analyticsCoordinator: AnalyticsCoordinator?
-    lazy private var etherKeystore = try? EtherKeystore(analyticsCoordinator: analyticsCoordinator)
     weak var delegate: AccountsViewControllerDelegate?
     var allowsAccountDeletion: Bool = false
     var hasWallets: Bool {
         return !keystore.wallets.isEmpty
     }
 
-    init(config: Config, keystore: Keystore, balanceCoordinator: GetNativeCryptoCurrencyBalanceCoordinator, analyticsCoordinator: AnalyticsCoordinator?) {
+    init(config: Config, keystore: Keystore, balanceCoordinator: GetNativeCryptoCurrencyBalanceCoordinator) {
         self.config = config
         self.keystore = keystore
         self.balanceCoordinator = balanceCoordinator
-        self.analyticsCoordinator = analyticsCoordinator
```

Comment:
@hboon why is this in the PR?

Reply:
Debated if I should include it. It was removed because it was added to provide wallet/keystore access. Not needed anymore because of this PR.